TrainingApp/Controller/SettingsViewController.swift

```diff
@@ -119,6 +174,87 @@ final class SettingsViewController: UIViewController {
     
     // MARK: - Helpers
     
+    private func checkThemeSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_THEME)) {
+            darkModeSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.DARK_THEME, forKey: self.THEME_KEY)
+        }
+        else {
+            darkModeSwitch.setOn(false, animated: true)
+            self.userDefaults.set(self.LIGHT_THEME, forKey: self.THEME_KEY)
+        }
+    }
+    
+    private func checkNotificationSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_NOTIFICATION)) {
+            notificationsSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.NOTIFICATION_ON, forKey: self.NOTIFICATION_KEY)
+            checkFlightSwitchState()
+            checkExecutiveSwitchState()
+            checkDiscountSwitchState()
+        }
+        else {
+            notificationsSwitch.setOn(false, animated: false)
+            self.userDefaults.set(self.NOTIFICATION_OFF, forKey: self.NOTIFICATION_KEY)
+            checkFlightSwitchState()
+            checkExecutiveSwitchState()
+            checkDiscountSwitchState()
+        }
+    }
+    
+    private func checkFlightSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_FLIGHT)) {
+            flightUpdatesSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.FLIGHT_ON, forKey: self.FLIGHT_KEY)
+        }
+        else {
+            flightUpdatesSwitch.setOn(false, animated: false)
+            self.userDefaults.set(self.FLIGHT_OFF, forKey: self.FLIGHT_KEY)
+        }
+    }
+    
+    private func checkExecutiveSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_EXECUTIVE)) {
+            executiveProgramSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.EXECUTIVE_ON, forKey: EXECUTIVE_KEY)
+        }
+        else {
+            executiveProgramSwitch.setOn(false, animated: false)
+            self.userDefaults.set(self.EXECUTIVE_OFF, forKey: EXECUTIVE_KEY)
+        }
+    }
+    
+    private func checkDiscountSwitchState() {
+        if(userDefaults.bool(forKey: ON_OFF_KEY_DISCOUNT)) {
+            discountDealsSwitch.setOn(true, animated: false)
+            self.userDefaults.set(self.DISCOUNT_ON, forKey: DISCOUNT_KEY)
+        }
+        else {
```

Comment:
remove this space
make it 
if {
} else {
}

Reply:
Done 👍